src/test/java/com/awesomeorg/airlineservice/integration_tests/ReservationControllerIntegrationTest.java

```diff
@@ -0,0 +1,119 @@
+package com.awesomeorg.airlineservice.integration_tests;
+
+import com.awesomeorg.airlineservice.AbstractIntegrationTest;
+import com.awesomeorg.airlineservice.entity.Passenger;
+import com.awesomeorg.airlineservice.entity.Reservation;
+import com.awesomeorg.airlineservice.protocol.CreatePassengerRequest;
+import com.awesomeorg.airlineservice.protocol.CreateReservationRequest;
+import com.awesomeorg.airlineservice.repository.PassengerRepository;
+import com.awesomeorg.airlineservice.repository.ReservationRepository;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+public class ReservationControllerIntegrationTest extends AbstractIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ReservationRepository reservationRepository;
+
+    @Autowired
+    private PassengerRepository passengerRepository;
+
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
You have a mapper in AbstractIntegrationTest, you don't need to declare it here as well.